Data::ODBC::Binder: virtual void Poco::Data::AbstractBinder::bind(...)’ was hidden

**Describe the bug**

bind functions hide virtual hide functions

**Logs**

Compiler output:

```
In file included from include/Poco/Data/ODBC/Binder.h:22,
                 from src/Binder.cpp:15:
/Users/matejk/git/alephone/devs/macchina.io/platform/Data/include/Poco/Data/AbstractBinder.h:332:22: warning: ‘virtual void Poco::Data::AbstractBinder::bind(std::size_t, const std::__cxx11::list<Poco::UUID>&, Direction)’ was hidden [-Woverloaded-virtual=]
  332 |         virtual void bind(std::size_t pos, const std::list<UUID>& val, Direction dir = PD_IN);
      |                      ^~~~
include/Poco/Data/ODBC/Binder.h:404:14: note:   by ‘Poco::Data::ODBC::Binder::bind’
  404 |         void bind(std::size_t pos, const char* const& pVal, Direction dir);
      |              ^~~~
/Users/matejk/git/alephone/devs/macchina.io/platform/Data/include/Poco/Data/AbstractBinder.h:329:22: warning: ‘virtual void Poco::Data::AbstractBinder::bind(std::size_t, const std::deque<Poco::UUID>&, Direction)’ was hidden [-Woverloaded-virtual=]
  329 |         virtual void bind(std::size_t pos, const std::deque<UUID>& val, Direction dir = PD_IN);
      |                      ^~~~
include/Poco/Data/ODBC/Binder.h:404:14: note:   by ‘Poco::Data::ODBC::Binder::bind’
  404 |         void bind(std::size_t pos, const char* const& pVal, Direction dir);
      |              ^~~~
/Users/matejk/git/alephone/devs/macchina.io/platform/Data/include/Poco/Data/AbstractBinder.h:326:22: warning: ‘virtual void Poco::Data::AbstractBinder::bind(std::size_t, const std::vector<Poco::UUID>&, Direction)’ was hidden [-Woverloaded-virtual=]
  326 |         virtual void bind(std::size_t pos, const std::vector<UUID>& val, Direction dir = PD_IN);
      |                      ^~~~
include/Poco/Data/ODBC/Binder.h:404:14: note:   by ‘Poco::Data::ODBC::Binder::bind’
  404 |         void bind(std::size_t pos, const char* const& pVal, Direction dir);
      |              ^~~~

```
**Please add relevant environment information:**
 - OS Type and Version: Ubuntu Linux 24.10
 - POCO Version: 1.14.0

--- a/Data/MySQL/include/Poco/Data/MySQL/Binder.h
+++ b/Data/MySQL/include/Poco/Data/MySQL/Binder.h
@@ -34,6 +34,8 @@ class MySQL_API Binder: public Poco::Data::AbstractBinder
 	/// Binds placeholders in the sql query to the provided values. Performs data types mapping.
 {
 public:
+	using AbstractBinder::bind;
+
 	typedef SharedPtr<Binder> Ptr;
 
 	Binder();

--- a/Data/ODBC/include/Poco/Data/ODBC/Binder.h
+++ b/Data/ODBC/include/Poco/Data/ODBC/Binder.h
@@ -57,6 +57,8 @@ class ODBC_API Binder: public Poco::Data::AbstractBinder
 	/// Binds placeholders in the sql query to the provided values. Performs data types mapping.
 {
 public:
+	using AbstractBinder::bind;
+
 	typedef AbstractBinder::Direction Direction;
 	typedef std::map<SQLPOINTER, SQLLEN> ParamMap;
 

--- a/Data/PostgreSQL/include/Poco/Data/PostgreSQL/Binder.h
+++ b/Data/PostgreSQL/include/Poco/Data/PostgreSQL/Binder.h
@@ -38,6 +38,8 @@ class PostgreSQL_API Binder: public Poco::Data::AbstractBinder
 	/// Allows data type mapping at statement execution time.
 {
 public:
+	using AbstractBinder::bind;
+
 	using Ptr = SharedPtr<Binder>;
 
 	Binder();

--- a/Data/SQLite/include/Poco/Data/SQLite/Binder.h
+++ b/Data/SQLite/include/Poco/Data/SQLite/Binder.h
@@ -35,6 +35,8 @@ class SQLite_API Binder: public Poco::Data::AbstractBinder
 	/// Binds placeholders in the sql query to the provided values. Performs data types mapping.
 {
 public:
+	using AbstractBinder::bind;
+
 	Binder(sqlite3_stmt* pStmt);
 		/// Creates the Binder.
 

